### PR TITLE
Differential Team (without version control)

### PR DESCRIPTION
**Pull request by rlee287**
Does not include version control (will merge that in separately once #214 is reviewed)

### CODE REVIEW

**Comment by agalperina** [comment]
Action endpoints are working fine on my end.

**Comment by thuang814** [comment]
image
Check these functions in app.js as well as the corresponding functions in queries to make sure they match.
I am currently getting this error when I run the put request for scenarios/stakeholders
I can't test out scenarios/stakeholders/history until this is fixed
image

**Comment by thuang814** [comment]
image
I am also getting this error when calling scenarios/stakeholders/conversation

File: goon-universe/queries.js [hunk 1]
```diff
@@ -863,6 +892,161 @@ function loadScenarioCSV(scenario_csv_string){
...
+        client.release();
+    }
+}
+function getInitActionResponse(submissionID, questionID, callback){
+    getMCQResponse(INIT_ACTION,submissionID, questionID).then((result) => callback(result));
+}
+
+function getFinalActionResponse(submissionID, questionID, callback){
+    getMCQResponse(FINAL_ACTION,submissionID, questionID).then((result) => callback(result));
+}
```
Comment (rasdy) [inline goon-universe/queries.js]: These functions are being called in app.js as `getInitActionResponse(studentID, scenarioID, function(result)`

Are there query functions that app.js can use to get the submissionID and questionID? If not, this should be re-written to take in a studentID and scenarioID.

**Review by rasdy — CHANGES_REQUESTED** [review]

**Comment by rlee287** [comment]
Notes from @thuang814 : get functions fixed but add function is still broken

**Review by aj8uppal — APPROVED** [review]

**Review by thuang814 — APPROVED** [review]

**Review by rasdy — APPROVED** [review]

**Pull request merged**